src/System.Runtime.Analyzers/VisualBasic/System.Runtime.VisualBasic.Analyzers.vbproj

```diff
@@ -116,8 +116,8 @@
     <Compile Include="BasicRethrowToPreserveStackDetails.Fixer.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.Fixer.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.Fixer.vb" />
+    <Compile Include="BasicInitializeStaticFieldsInline.vb" />
```

Comment:
I think naming convention and style is an issue we must agree on across all the analyzer implementations, thus not suitable to discuss here in a PR. I have create issue https://github.com/dotnet/roslyn-analyzers/issues/347 for this, please have all the discussion over there, and hopefully we can get a finalized documentation on this.

@michaelcfanning @lgolding @srivatsn @mavasani